### DISCUSSION TITLE
plugin documentation: link doesnt work.

### DISCUSSION THREAD

**Discussion by ukrishne** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter#89] [Q&A] [answered]
Dear Team,

plugin documentation is taking us to SBB portal where public is not having permission. can you please make it available for public.
![image](https://github.com/user-attachments/assets/2de9cc73-6c5c-4824-8c16-7166a912feec)

thanks in Advance

**Comment by ralfsmolarek3** [comment]
There is not much additional info in that document. We will transfer all info into git (https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/issues/97) but until that point in time I attache the latest version here
[Polarion Extensions Specification PDF Exporter.pdf](https://github.com/user-attachments/files/16361386/Polarion.Extensions.Specification.PDF.Exporter.pdf)

**Reply by ukrishne** [reply]
thanks ralf.

**Comment by grigoriev** [comment, marked as answer]
Hi @ukrishne,

link in readme file has been fixed:

- [PDF document](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/blob/main/docs/pdf-exporter.pdf) is placed under Git control
- [Open API Spec](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/blob/main/docs/openapi.json) added as well

**Reply by ukrishne** [reply]
many thanks. works well. great job
